mysql/datadog_checks/mysql/queries.py

```diff
@@ -17,6 +17,11 @@
 FROM     information_schema.tables
 GROUP BY table_schema"""
 
+SQL_QUERY_TABLE_SIZE = """\
+SELECT   table_name, IFNULL(SUM(data_length+index_length)/1024/1024,0) AS total_mb
+FROM     information_schema.tables
+GROUP BY table_name"""
```

Comment:
With this query, if we have 2 tables with the same name in different schema, won't it add them up ?
(cf the `GROUP BY table_name`)